src/core/ComboBox/ComboBox.test.tsx

```diff
@@ -59,12 +64,185 @@ it('should render with selected value', () => {
 
   input.focus();
   container.querySelectorAll('.iui-menu-item').forEach((item, index) => {
-    expect(item).toHaveTextContent(`Item ${index + 1}`);
-    if (index === 1) {
+    expect(item).toHaveTextContent(`Item ${index}`);
+    if (index === 2) {
```

Comment:
You could also check that others won't have the class and attribute.

Reply:
Is it really worth it? I already check for normal state in the basic test